OCR/ocr/assets/labels.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
copied these into the main service directory as well but not necessarily sure how I feel about these being in two different places even though I know they might not even be located in the repo in the future anyways but if we do want to only use them from the og test directory, I will need to move the `pyinstaller.py` function to the root dir.